source/blog/2017-02-25-oVirt-gamification.md

```diff
@@ -1,53 +1,52 @@
 
 ### So What is New?
-The oVirt infra team recently reached out to 'GetBadges', a company which provides 'Gamifcation as a Service'.
-Luckly for us, open source projects gets to have a free game! So oVirt was rewarded with its own [oVirt Open Source Game](https://ovirt-ovirt-engine.getbadges.io/activity)
-The game works automagically every time you contribute to the project, current activated integrations are [3]: 
- * GitHub  - any commit merged rewards points 
- * Jenkins - any successful build rewards points and failed builds deduct points 
- * Jira    - any new ticket opened or closed in jira.ovirt.org rewards points
 
-Other than points, you are rewarded with cool badges every time you acomplish an achievement, like triggeing a first successful build. 
-There are more cool features like slaying monsters :), you can read more detailed info on the links below.
+The oVirt infra team recently reached out to 'GetBadges,' a company which provides 'Gamifcation as a Service'.
+Luckly for us, open source projects gets to have a free game! So oVirt was rewarded with its own [oVirt Open Source Game](https://ovirt-ovirt-engine.getbadges.io/activity).
 
-### oVirt 2016 game stats
-So you might ask yourself how are you doing so far in the game.. <br>
-Checkout the 2016 statistics shown in the infographic below. 
-[oVirt Game 2016 Statistics](/images/oVirt-game-stats.png)
+The game works automagically every time you contribute to the project. Current integrations are only active on specific projects like 'ovirt-engine' and 'vdsm'. If you wish to add your project to the game, please reach out to oVirt infra team and ask to add it. Current activation channels are: 
 
+    * GitHub  - any commit merged rewards points 
+    * Jenkins - any successful build rewards points and failed builds deduct points 
+    * Jira    - any new ticket opened or closed in jira.ovirt.org rewards points
 
-## More Info on GetBadges and the Game Elements
-You can find more info and details on the GetBadges game on the GetBadges blog site.
+Other than points, you are rewarded with cool badges every time you acomplish an achievement, like triggering a first successful build. There are more cool features like slaying monsters :), you can read more detailed info on the GetBadges links below.
 
-* [FAQ Part I](https://getbadges.io/blog/frequently-asked-questions-1)<br>
-* [FAQ Part II](https://getbadges.io/blog/frequently-asked-questions-pt-2)<br>
-* [GetBadges Blog](https://getbadges.io/blog/)<br>
-* [Understanding Badges](https://getbadges.io/blog/understanding-badges-in-our-it-gamification-platform)<br>
-* [How to motivate developers](http://getbadges.io/blog/how-to-motivate-developers)<br>
+### oVirt 2016 Game Stats
 
+So you might ask yourself how are you doing so far in the game. Checkout the 2016 statistics shown in the infographic below. 
 
-## References
+![oVirt Game 2016 Statistics](/images/oVirt-game-stats.png)
```

Comment:
@bproffitt  is it possible to show this image as part of the blog instead of link to external image?